themes-default/slim/views/displayShow.mako

```diff
@@ -11,10 +11,10 @@
     from medusa.indexers.utils import mappings
 %>
 <%block name="scripts">
-<script type="text/javascript" src="js/plot-tooltip.js?${sbPID}"></script>
 <script type="text/javascript" src="js/rating-tooltip.js?${sbPID}"></script>
 <script type="text/javascript" src="js/ajax-episode-search.js?${sbPID}"></script>
 <script type="text/javascript" src="js/ajax-episode-subtitles.js?${sbPID}"></script>
+<%include file="/vue-components/plot-info.mako"/>
```

Comment:
The template isn't loaded by js at all unless it's used. I wanted to add it to the `main.mako` since I played around with vuex and vue-router and noticed we needed access to all components from `main.mako`.